tests/src/Recipe/RecipeEntity.php

```diff
@@ -74,18 +74,16 @@ public function testMalformedRecipeEntityGenerateHTML()
      *
      * @return void
      */
-    public function testSuccessRecipeEntity()
-    {
-        $testIngredient = new GRUB\RecipeEntity ('string', 'string', 'string', 'string');
-        $result = $testIngredient->generateHTML();
+    public function testSuccessRecipeEntityGenerateHTMLSaved(){
+        $testIngredient = new GRUB\Recipe\RecipeEntity ('string','string','string','string');
+        $result = $testIngredient->GenerateHTMLSaved();
         $expected = "<div class='recipe'>";
         $expected .= "<div class='left'>";
         $expected .= "<div class='recipeImage'>";
         $expected .= "<img src='string'/>";
         $expected .= "</div>";
-        $expected .= "<a href='string'>";
-        $expected .= "<button>Link to recipe</button>";
-        $expected .= "</a>";
+        $expected .= "<a name='deleteRecipe'><button>Delete Recipe</button></a>";
```

Comment:
Needs hidden form, see marcs addition to recipeEntitiy

Reply:
Fixed